src/clarity-angular/emphasis/alert/alert.html

```diff
@@ -1,17 +1,19 @@
 <!--
-  ~ Copyright (c) 2016 VMware, Inc. All Rights Reserved.
+  ~ Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
   ~ This software is released under MIT license.
   ~ The full license information can be found in LICENSE in the root directory of this project.
   -->
 
 <div
     *ngIf="!_closed"
     class="alert"
-    [ngClass]="alertType"
+    [ngClass]="alertClass"
     [class.alert-sm]="isSmall"
     [class.alert-app-level]="isAppLevel">
+    <div class="alert-items">
+        <ng-content></ng-content>
+    </div>
     <button type="button" class="close" aria-label="Close" *ngIf="closable" (click)="close()">
         <clr-icon aria-hidden="true" shape="close"></clr-icon>
     </button>
```

Comment:
This might be a stupid question, but does the order really matter? We could simply use `flex-order: 100` on the `.close` button and it would always be the last element in the flexbox, no matter the order of the actual DOM.